data/meson.build

```diff
@@ -54,6 +54,7 @@ svgs = [
 	'svgs/logitech-g9.svg',
 	'svgs/logitech-g-pro.svg',
 	'svgs/logitech-g-pro-wireless.svg',
+	'svgs/logitech-lightspeed.svg',
```

Comment:
That's not enough anymore now that the SVGs must be handled  by piper. The lookup table is in `data/svgs/svg-lookup.ini` and the only info we get is `usb:046d:c539` from ratbagd. So we'll look up that string in the .ini file and come up with the `logitech-g900.svg` file. So that symlink doesn't do anything bug exercise the file system on install a bit. Which is outside of piper's scope ;)

You'll need to update the .ini file together with any file changes.